From db69ccb17400702a36a55be149ce62bd2fa25935 Mon Sep 17 00:00:00 2001
From: Daniel Baluta <daniel.baluta@nxp.com>
Date: Thu, 4 Oct 2018 11:14:02 +0300
Subject: [PATCH 4832/5242] MLK-18497-1: ASoC: fsl: dsp_proxy: Fix license
 headers

commit  de0fb4c5d3e02645be63e2908a0cb9af6543c787 from
https://source.codeaurora.org/external/imx/linux-imx.git

This is a cleanup patch to have all the headers style
consistent across DSP driver.

Reviewed-by: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com
Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_dsp_proxy.c |   20 +++++---------------
 sound/soc/fsl/fsl_dsp_proxy.h |   15 +++++----------
 2 files changed, 10 insertions(+), 25 deletions(-)

diff --git a/sound/soc/fsl/fsl_dsp_proxy.c b/sound/soc/fsl/fsl_dsp_proxy.c
index 5bce995..30e5f81 100644
--- a/sound/soc/fsl/fsl_dsp_proxy.c
+++ b/sound/soc/fsl/fsl_dsp_proxy.c
@@ -1,24 +1,14 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR MIT)
-/*******************************************************************************
- *
- * Copyright (C) 2017 Cadence Design Systems, Inc.
- * Copyright 2018 NXP
- *
- ******************************************************************************/
-/*******************************************************************************
- * fsl_dsp_proxy.c
- *
- * DSP proxy driver
- *
- * DSP proxy driver is used to transfer messages between dsp driver
- * and dsp framework
- ******************************************************************************/
+//
+// DSP proxy driver transfers messages between DSP driver and DSP framework
+//
+// Copyright 2018 NXP
+// Copyright (C) 2017 Cadence Design Systems, Inc.
 
 #include <soc/imx8/sc/ipc.h>
 #include "fsl_dsp_proxy.h"
 #include "fsl_dsp.h"
 
-
 /* ...initialize message queue */
 void xf_msg_queue_init(struct xf_msg_queue *queue)
 {
diff --git a/sound/soc/fsl/fsl_dsp_proxy.h b/sound/soc/fsl/fsl_dsp_proxy.h
index 0942b8d..131acf3 100644
--- a/sound/soc/fsl/fsl_dsp_proxy.h
+++ b/sound/soc/fsl/fsl_dsp_proxy.h
@@ -1,15 +1,10 @@
-/* SPDX-License-Identifier: (GPL-2.0+ OR MIT)*/
-/*******************************************************************************
+/* SPDX-License-Identifier: (GPL-2.0+ OR MIT) */
+/*
+ * DSP proxy header - commands/responses from DSP driver to DSP ramework
  *
- * Copyright (c) 2017 Cadence Design Systems, Inc.
  * Copyright 2018 NXP
- *
- ************************************************************/
-/************************************************************
- * fsl_dsp_proxy.h
- *
- * Proxy commmand/response messages
- ************************************************************/
+ * Copyright (c) 2017 Cadence Design Systems, Inc.
+ */
 
 #ifndef __FSL_DSP_PROXY_H
 #define __FSL_DSP_PROXY_H
-- 
1.7.9.5

